internals/secrethub/service_gcp_init.go

```diff
@@ -108,6 +109,31 @@ func (cmd *ServiceGCPInitCommand) Run() error {
 		cmd.description = "GCP Service Account " + roleNameFromRole(cmd.serviceAccountEmail)
 	}
 
+	projectID, err := api.ProjectIDFromGCPEmail(cmd.serviceAccountEmail)
+	if err != nil {
+		return fmt.Errorf("invalid service account email: %s", err)
+	}
+
+	_, err = client.IDPLinks().GCP().Get(cmd.repo.GetNamespace(), projectID)
+	if api.IsErrNotFound(err) {
+		fmt.Fprintf(cmd.io.Output(), "GCP project %s is not yet linked to the namespace %s. ", projectID, cmd.repo.GetNamespace())
```

Comment:
Do you think we should explain what it means for a project to be linked to the namespace and what are the consequences?

For example: Is this a one-on-one link, can I still create other service accounts for this project in other namespaces?

I think we want to explain to the user why they want to answer yes in most cases, but also, why they might want to answer no in other cases.

Reply:
Added some more text